models/project/project.go

```diff
@@ -163,13 +166,15 @@ func (u *ProjectStruct) Save() bool {
 	return true
 }
 
-func (u *ProjectStruct) Update(title string, description string) bool {
+func (u *ProjectStruct) Update(title string, description string, showIssues bool, showPullRequests bool) bool {
```

Comment:
__[golint]__
* [golint] exported method ProjectStruct.Update should have comment or be unexported